pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -235,6 +236,10 @@ func (o *ReconcileSCCOptions) computeUpdatedSCC(expected kapi.SecurityContextCon
 		if actual.Priority != nil {
 			expected.Priority = actual.Priority
 		}
+
+		// preserve labels and annotations
+		util.AddObjectLabels(&expected, actual.Labels)
```

Comment:
also, I'd rather have a function that takes two map[string]string and returns a new one, not one that modifies one of it's args in place